docs/modules/plugins/partials/generic-ui-steps.adoc

```diff
@@ -80,6 +80,25 @@ When I wait until number of elements located by `tagName(img)` is equal to `5`
 When I wait until number of elements located by `tagName(img):1` is equal to `5`
 ----
 
+
+=== Wait for element to stop moving
+
+Waits for element with the specified locator to stop moving.
```

Comment:
we also need to define what "moving" means? in other words is animated GIF considered "moving"?

Reply:
added Note